.github/workflows/ci.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.19.x, 1.20.x, 1.21.x]
```

Comment:
The `go.mod` already indicated this needs Go 1.20. And I'm using some stuff from `os/exec` that was added in Go 1.20. So dropping 1.19.